druid-shell/src/window.rs

```diff
@@ -173,7 +173,7 @@ impl WindowBuilder {
     ///
     /// If this fails, your application should exit.
     pub fn build(self) -> Result<WindowHandle, Error> {
-        self.0.build().map(WindowHandle)
+        self.0.build().map(WindowHandle).map_err(Into::into)
```

Comment:
I wonder if this would be slightly cleaner with a `?` formulation. But that's strictly a style question.

Reply:
Good question, I don't have strong feelings. This feels less magic and more functional, `Ok(thing()?)` always seems a bit awkward to me.